packages/docsify-dark-mode/README.md

```diff
@@ -1,25 +1,44 @@
 # docsify-dark-mode
 
-This is docsify plugin which enables dark mode for your docsify site
+[![JavaScript Style Guide](https://badgen.net/badge/code%20style/standard/f2a)](https://github.com/standard/standard)
+[![jsDelivr hits (npm)](https://badgen.net/jsdelivr/hits/npm/docsify-dark-mode)](https://www.jsdelivr.com/package/npm/docsify-dark-mode)
 
-[![JavaScript Style Guide](https://cdn.rawgit.com/standard/standard/master/badge.svg)](https://github.com/standard/standard)
-[![](https://data.jsdelivr.com/v1/package/npm/docsify-dark-mode/badge)](https://www.jsdelivr.com/package/npm/docsify-dark-mode)
```

Comment:
Hey ! can you please add the old badges. It kind of looks cool than round badges. Else every thing looks perfect. 👍

Reply:
I just reverted the changes.